addons/website/models/website_visitor.py

```diff
@@ -327,6 +327,8 @@ def _merge_visitor(self, target):
 
         :param target: main visitor, target of link process;
         """
+        if self == target:
```

Comment:
it means that all override of `_merge_visitor` will be called
shouldn't we do this verification before calling the method ?

Reply:
ok for me :)